mscore/editstafftype.ui

```diff
@@ -651,6 +651,13 @@
                    </widget>
                   </item>
                   <item>
+                   <widget class="QCheckBox" name="showTabFingering">
+                    <property name="text">
+                     <string>Show fingerings</string>
```

Comment:
In 2.1, indeed it is show fingerings, as they are handled, but not shown. In master it probably would be better to say "use (or allow) fingerings", but I wouldn't change the name of the variable (if, then only in the dev-version), as it is at least a sort of compatibility (and in the end it has the same effect ...). BTW - I named the string in the dialog "show fingerings", as this is it in 2.1!